### PR TITLE
Example which creates output for use with new Skill Builder

### PR DESCRIPTION
**Pull request by funkydan2**
Putting <%=app.schemas.skillBuilder()%> in an ejs file seems to create the output which can be used with the new Skill Builder.

(Sorry, I'm not super familiar with using GitHub, hopefully this PR is in the right format to be useful.)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 * [#315](https://github.com/alexa-js/alexa-app/pull/315): Fix card tests and required card type - [@kobim](https://github.com/kobim).
 * [#314](https://github.com/alexa-js/alexa-app/pull/314): Fix handling of `undefined` slot values - [@User1m](https://github.com/user1m).
+* Included Skill Builder example in example.ejs - [@funkydan2](https://github.com/funkydan2)
```
Comment (dblock) [inline CHANGELOG.md]: Can you please make it look like the other changes here? Include the PR number and a period in the end.

**Comment by funkydan2** [comment]
Sorry, @dblock is this closer to the formatting required?

**Comment by dblock** [comment]
Merged, thanks.

**Comment by matt-kruse** [comment]
Just catching up to this, but I've been using this in my "deploy" script and it works well:

node -e "console.log(require('./src/index.js').app.schemas.askcli())" > models\en-US.json

**Pull request merged**